server/tests/test_db.py

```diff
@@ -532,61 +533,77 @@ async def test_migration_v5(self):
     async def test_migration_v6(self):
         await migrations.execute_migrations(self.c, self.dbconn.config, 5)
 
-        with patch('uuid.uuid4') as mock_uuid:
-            mock_uuid.return_value = '1'
-            winner = await self.dbconn.authenticate_user('test_user_1', b'password', auto_create=True)
-
-            mock_uuid.return_value = '2'
-            loser = await self.dbconn.authenticate_user('test_user_2', b'password', auto_create=True)
-
-            mock_uuid.return_value = '1234'
-            test_match = DbMatchReport(
-                winner_id=winner.user_id,
-                loser_id=loser.user_id,
-                winner_pieces_left=10,
-                loser_pieces_left=5,
-                move_counter=20,
-                grid_size='small',
-                squadron_size='medium',
-                started_at=datetime(2020, 1, 1, 0, 0, 0, tzinfo=timezone.utc),
-                finished_at=datetime(2020, 1, 1, 1, 0, 0, tzinfo=timezone.utc),
-                is_ranked=True,
-                is_void=False,
+        async def make_user(id_: str, username: str) -> DbUser:
+            await self.c.execute(
+                "insert or ignore into users(id, username, password, created_at, discord_user_id"
+                ") values (?, ?, ?, ?, ?)",
+                (id_, username, password_hash(b'password'), datetime.now(timezone.utc).timestamp(), None)
             )
             await self.c.execute(
```

Comment:
1. Why selecting the same things that were inserted at the line above?
2. Do we need this data at all? If I see correctly, we're only using the ID.

Reply:
mb - we do need to create these users but we don't need to query for them. we can just use raw ids in test